Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -881,18 +900,26 @@ def fetch_incidents():
     demisto.info("Executing Prisma Cloud (RedLock) fetch_incidents with payload: {}".format(payload))
     response = req('POST', 'alert', payload, {'detailed': 'true'})
     incidents = []
+
+    fetched_ids[now] = []
+
     for alert in response:
-        if any(alert.get('id') in x for x in fetched_ids):
-            demisto.debug(f"Fetched {alert.get('id')} already. Skipping")
+        alert_id = alert.get('id')
+        if any(alert_id in existing_fetched_ids for existing_fetched_ids in fetched_ids.values()):
+            demisto.debug(f"Fetched {alert_id} already. Skipping")
             continue
-        demisto.debug(f"{alert.get('id')} has not been fetched. Processing.")
+
+        demisto.debug(f"{alert_id} has not been fetched. Processing.")
```

Comment:
```suggestion
        demisto.debug(f"Processing new fetched alert {alert_id}.")
```
Just because the previous log confused customers (they thought it is not fetched at all, not that it is fetched now), this is the message we use in V2.